src/runtime/arch/s390x-options.mk

```diff
@@ -11,3 +11,10 @@ MACHINEACCELERATORS :=
 CPUFEATURES :=
 
 QEMUCMD := qemu-system-s390x
+
+# See https://github.com/kata-containers/osbuilder/issues/217
+FEDORA_LIKE = $(shell grep fedora /etc/os-release 2> /dev/null)
```

Comment:
Might be overkill but you could anchor this with `... egrep "\<fedora\>" ...`.

Reply:
This is going to match CentOS, AlmaLinux, Rocky Linux, RHEL, OEL, etc.
I guess this is **exactly** what you're trying to do, but I'd like a confirmation from you, @Jakob-Naucke.

Reply:
> Might be overkill but you could anchor this with `... egrep "\<fedora\>" ...`.

Updated.

> This is going to match CentOS, AlmaLinux, Rocky Linux, RHEL, OEL, etc.
> I guess this is *exactly* what you're trying to do, but I'd like a confirmation from you, @Jakob-Naucke.

Most of those don't package for s390x :stuck_out_tongue_winking_eye: but I also just tried this on ClefOS (~ s390x's CentOS) and it needs it too. So yes.